### PR TITLE
Filter additional props for the PanelViewports component

### PR DESCRIPTION
**Pull request by mahdiyazdani**
This PR introduces a new component-specific filter to allow modifying additional properties depending on the viewport panel component that is currently active.

### CODE REVIEW

**Review by gooklani — APPROVED** [review]
Thank you @mahdiyazdani 🙌

File: src/media-upload/index.js [hunk 1]
```diff
@@ -76,7 +80,7 @@ import Constants from './constants';
  *     value={ id }
  * />
  */
-function MediaUpload( { onChange, value, ...otherProps } ) {
+function MediaUpload( { onChange, noticeOperations, noticeUI, value, ...otherProps } ) {
```
Comment (kuserich) [inline src/media-upload/index.js:83]: Changing the order of these parameters potentially introduces a breaking change for blocks that use this component. Do you know if this is currently the case?

**Review by kuserich — APPROVED** [review]
Thank you, @mahdiyazdani . These changes might break some of our blocks. Have you checked whether this could be an issue?

File: src/media-upload/index.js (hunk 1 above)
Comment (mahdiyazdani) [reply src/media-upload/index.js:83]: In this proposal, the exported component is wrapped with the “withNotices” higher-order component that encapsulates the original component with the additional props “noticeOperations” and “noticeUI”. These properties would always have values, and the change in order wouldn’t affect the current usage of this component elsewhere. 🙂 

Additionally, I have published a beta version for testing purposes:
```bash 
npm i @sixa/wp-block-components@1.14.1-beta.0
```

Comment (kuserich) [reply src/media-upload/index.js:83]: ah, yes, I see. Thank you for elaborating and for preparing a beta version, @mahdiyazdani

**Pull request merged**